### DISCUSSION TITLE
How can we make the color more transparent, making the boundaries more clear

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#2018] [Ask Anything]
Hi, everyone,
 I have some EBSD data and want to make some plots. How can we make the colors of grains more transparent, so that we can see the boundaries more clearly?
![image](https://github.com/mtex-toolbox/mtex/assets/69452222/b5aa9060-d4c1-4d36-ba51-b752e6e886b4)
Best regards,
Brian

**Comment by zmichels** [comment]
I would try 'FaceAlpha' flag for translucency.

And you can adjust the 'LineWidth' and color of the plot to make grain boundaries more prominent.

**Reply by Brian-Liu-rx** [reply]
FaceAlpha works! Thanks!

**Reply by Brian-Liu-rx** [reply]
Yes, it is not a good idea to change the color, since it represents the Millers
